examples/substrait-spark/app/src/main/java/io/substrait/examples/SparkDataset.java

```diff
@@ -0,0 +1,80 @@
+package io.substrait.examples;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan;
+import java.io.IOException;
+import java.nio.file.*;
```

Comment:
minor: we generally avoid * imports in this repo

Reply:
sure.. no problem;